build.install4j

```diff
@@ -856,9 +852,7 @@ return true;</string>
       <excludedLaunchers />
       <excludedBeans />
       <overriddenPrincipalLanguage id="en" customLocalizationFile="" />
-      <exclude>
-        <entry location="build/assets/icons/icon.icns" fileType="regular" />
-      </exclude>
```

Comment:
This entry (and the two identical ones below) is basically a no-op.  install4j expects the entry location to be relative to the installation directory at runtime, not the root of the build directory.  That is, this entry should have had a location of `icons/icon.icns` if it was intended to do anything.  (I confirmed that after running the Unix installer from `master` that the `icons/icon.icns` file is still present, thus proving this entry is useless.)